### PR TITLE
Update hyprlock.conf

### PR DESCRIPTION
**Pull request by TechLionDev**
Fixes Rendering Issue #3

### CODE REVIEW

File: hyprlock.conf [hunk 1]
```diff
@@ -12,9 +14,12 @@ general {
...
 # BACKGROUND
 background {
+    path = screenshot
+    blur_size = 5
+    blur_passes = 4
     monitor =
-    path = ~/.config/background
-    blur_passes = 0
+    # path = ~/.config/background
+    # blur_passes = 0
```
Comment (NikSneMC) [inline hyprlock.conf:22]: I think that should be untouched

**Review by NikSneMC — CHANGES_REQUESTED** [review]
Use mauve, not sapphire (IDK when style guide will be updated to mention mauve as default color)

**Review by NikSneMC — APPROVED** [review]

**Comment by NikSneMC** [comment]
@jaybobsilver does [this](https://github.com/TechLionDev/hyprlock/blob/main/hyprlock.conf) config file work for you?

**Comment by jaybobsilver** [comment]
I replaced my current config with that, and the placeholder text simply does not appear. It is replaced with the generic <i>Input Password</i> placeholder text. Removing the tags around the text does make the correct text appear, however.

**Comment by jaybobsilver** [comment]
I replaced my current config with that, and the placeholder text does not appear. It is replaced with the generic <i>Input Password</i> text. However, when I apply the changes to my own config the proper text appears. A quick glance between the two and I'm still unsure why. Here is my whole config:

```
source = $HOME/.config/hypr/colorscheme/macchiato.conf

$accent = $lavender
$accent2 = $lavenderAlpha
$text2 = $textAlpha
$text = $text
$font = JetBrainsMono Nerd Font

# GENERAL
general {
    disable_loading_bar = true
    hide_cursor = true
    grace = 300
}

# BACKGROUND
background {
    monitor =
    path = $HOME/Pictures/Wallpapers/SMACS0723_3840x2160.png # .png only for now...
    blur_passes = 0
    color = $base
# All these options are taken from hyprland, see https://wiki.hyprland.org/Configuring/Variables/#blur for explanations
    blur_passes = 2 # 0 disables blurring
    blur_size = 2
    noise = 0.0117
    contrast = 0.8916
    brightness = 0.8172
    vibrancy = 0.1696
    vibrancy_darkness = 0.0
}

# TIME
label {
    monitor =
    text = cmd[update:30] echo "$(date +"%r")"
    color = $text
    font_size = 60
    font_family = $font
    position = -30, 0
    halign = right
    valign = top
}

# DATE 
label {
    monitor = 
    text = cmd[update:43200000] echo "$(date +"%a, %b. %d %Y")"
    color = $text
    font_size = 20
    font_family = $font
    position = -30, -100
    halign = right
    valign = top
}

# USER AVATAR

image {
    monitor = 
    path = $HOME/.face
    size = 100
    border_color = $accent
    position = 0, 75
    halign = center
    valign = center
}

# INPUT FIELD
input-field {
    monitor =
    size = 300, 60
    outline_thickness = 2
    dots_size = 0.2
    dots_spacing = 0.15
    dots_center = true
    outer_color = $accent
    inner_color = $surface0
    font_color = rgb(cad3f5)
    fade_on_empty = false
    fade_timeout = 1000 # in ms
   # placeholder_text = <span foreground="##cad3f5"><i>󰌾 Logged in as </i><span foreground="##b7bdf8">$USER</span></span>
   placeholder_text = <span foreground="##$text2"><i>󰌾 Logged in as </i><span foreground="##$accent2">$USER</span></span>
    hide_input = false
    check_color = $accent
    fail_color = $red
    fail_transition = 300
    fail_text = <i>$FAIL <b>($ATTEMPTS)</b></i>
    capslock_color = $yellow
    numlock_color = $peach
    invert_numlock = true
    check_color = $green
    position = 0, -35
    halign = center
    valign = center
}

```

**Comment by TechLionDev** [comment]
so if u just completely replace it is broken, yet if u manually change the lines its not? is that what you meant by

> I replaced my current config with that, and the placeholder text does not appear. It is replaced with the generic Input Password text. However, when I apply the changes to my own config the proper text appears.

**Comment by NikSneMC** [comment]
Maybe I should use whiskers to generate all files automatically. Like this
image

**Comment by NikSneMC** [comment]
> I replaced my current config with that, and the placeholder text does not appear. It is replaced with the generic Input Password text. However, when I apply the changes to my own config the proper text appears. A quick glance between the two and I'm still unsure why. Here is my whole config:
> 
> ```
> source = $HOME/.config/hypr/colorscheme/macchiato.conf
> 
> $accent = $lavender
> $accent2 = $lavenderAlpha
> $text2 = $textAlpha
> $text = $text
> $font = JetBrainsMono Nerd Font
> 
> # GENERAL
> general {
>     disable_loading_bar = true
>     hide_cursor = true
>     grace = 300
> }
> 
> # BACKGROUND
> background {
>     monitor =
>     path = $HOME/Pictures/Wallpapers/SMACS0723_3840x2160.png # .png only for now...
>     blur_passes = 0
>     color = $base
> # All these options are taken from hyprland, see https://wiki.hyprland.org/Configuring/Variables/#blur for explanations
>     blur_passes = 2 # 0 disables blurring
>     blur_size = 2
>     noise = 0.0117
>     contrast = 0.8916
>     brightness = 0.8172
>     vibrancy = 0.1696
>     vibrancy_darkness = 0.0
> }
> 
> # TIME
> label {
>     monitor =
>     text = cmd[update:30] echo "$(date +"%r")"
>     color = $text
>     font_size = 60
>     font_family = $font
>     position = -30, 0
>     halign = right
>     valign = top
> }
> 
> # DATE 
> label {
>     monitor = 
>     text = cmd[update:43200000] echo "$(date +"%a, %b. %d %Y")"
>     color = $text
>     font_size = 20
>     font_family = $font
>     position = -30, -100
>     halign = right
>     valign = top
> }
> 
> # USER AVATAR
> 
> image {
>     monitor = 
>     path = $HOME/.face
>     size = 100
>     border_color = $accent
>     position = 0, 75
>     halign = center
>     valign = center
> }
> 
> # INPUT FIELD
> input-field {
>     monitor =
>     size = 300, 60
>     outline_thickness = 2
>     dots_size = 0.2
>     dots_spacing = 0.15
>     dots_center = true
>     outer_color = $accent
>     inner_color = $surface0
>     font_color = rgb(cad3f5)
>     fade_on_empty = false
>     fade_timeout = 1000 # in ms
>    # placeholder_text = <span foreground="##cad3f5"><i>󰌾 Logged in as </i><span foreground="##b7bdf8">$USER</span></span>
>    placeholder_text = <span foreground="##$text2"><i>󰌾 Logged in as </i><span foreground="##$accent2">$USER</span></span>
>     hide_input = false
>     check_color = $accent
>     fail_color = $red
>     fail_transition = 300
>     fail_text = <i>$FAIL <b>($ATTEMPTS)</b></i>
>     capslock_color = $yellow
>     numlock_color = $peach
>     invert_numlock = true
>     check_color = $green
>     position = 0, -35
>     halign = center
>     valign = center
> }
> ```

Does this work?
```
$font = JetBrainsMono Nerd Font

# GENERAL
general {
    disable_loading_bar = true
    hide_cursor = true
}

# BACKGROUND
background {
    monitor =
    path = ~/.config/background
    blur_passes = 0
    color = $base
}

# TIME
label {
    monitor =
    text = cmd[update:30000] echo "$(date +"%R")"
    color = rgb(cad3f5)
    font_size = 90
    font_family = $font
    position = -30, 0
    halign = right
    valign = top
}

# DATE 
label {
    monitor = 
    text = cmd[update:43200000] echo "$(date +"%A, %d %B %Y")"
    color = rgb(cad3f5)
    font_size = 25
    font_family = $font
    position = -30, -150
    halign = right
    valign = top
}

# USER AVATAR

image {
    monitor = 
    path = ~/.face
    size = 100
    border_color = int(b7bdf8)

    position = 0, 75
    halign = center
    valign = center
}

# INPUT FIELD
input-field {
    monitor =
    size = 300, 60
    outline_thickness = 4
    dots_size = 0.2
    dots_spacing = 0.2
    dots_center = true
    outer_color = int(b7bdf8)
    inner_color = rgb(363a4f)
    font_color = rgb(cad3f5)
    fade_on_empty = false
    placeholder_text = <span foreground="##cad3f5"><i>󰌾 Logged in as </i><span foreground="##b7bdf8">$USER</span></span>
    hide_input = false
    check_color = rgb(b7bdf8)
    fail_color = rgb(ed8796)
    fail_text = <i>$FAIL <b>($ATTEMPTS)</b></i>
    capslock_color = rgb(eed49f)
    position = 0, -35
    halign = center
    valign = center
}
```

**Comment by TechLionDev** [comment]
Yes it Does, just it doesnt show the accent color as much as mine does

**Comment by NikSneMC** [comment]
So, maybe I'll use that version instead of theme file from Hyprland repo

**Pull request closed without merging**